design-proposals/high-level-plane.md

```diff
@@ -0,0 +1,57 @@
+# High-level plane
+
+Author: Fabian Deutsch \<fdeutsch@redhat.com\>
+
+## Introduction
+
+Today KubeVirt is exposing a pretty low-level interface to virtualization capabilities.
+
+But this low-level API might be too low-level for i.e. UIs to integrate. On the higher level its typically required to set policies, the lower layer then provides the mechanics to implement this policy. The suggested high-level plane is the middle layer, translating policies into the mechanics.
```

Comment:
To clarify this:  I don't like the term. Having a bunch of "add-ons", like kubernetes has helm or the dashboard sounds like a better description for me.